src/routes/studies/index.svelte

```diff
@@ -14,18 +14,26 @@
   const notifications: NotificationStore = getContext("rally:notifications");
 
   function joinStudy(studyId) {
+    const study = $store.studies.find((s) => s.studyId === studyId);
+
+    if (!study) {
+      throw new Error(`Study with id: ${studyId} not found.`);      
+    }
+
+    window.open(study.downloadLink);
```

Comment:
```suggestion
    window.open(study.downloadLink, "_blank");
```
This specify the `_blank` [target keyword](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target):

```
_blank: usually a new tab, but users can configure browsers to open a new window instead.
```